phoenix-core/src/it/java/org/apache/phoenix/end2end/ExplainPlanWithStatsEnabledIT.java

```diff
@@ -162,7 +162,7 @@ public void testBytesRowsForSelectWithLimitIgnored() throws Exception {
         List<Object> binds = Lists.newArrayList();
         try (Connection conn = DriverManager.getConnection(getUrl())) {
             Estimate info = getByteRowEstimates(conn, sql, binds);
-            assertEquals((Long) 691L, info.estimatedBytes);
+            assertEquals((Long) 979L, info.estimatedBytes);
```

Comment:
The entire test suite uses hard coded values. This test suite needs to be restructured so that these values are set using a formula. At this moment, this test suite is useless to me since the expected value is set to the actual value manually.